test/integration/test_connection_management.py

```diff
@@ -102,36 +106,56 @@ def test_http_h2_connection_management_single_request_per_conn_1(http_test_serve
   _connection_management_test_request_per_connection(http_test_server_fixture, 5, True)
 
 
-def test_h1_pool_strategy(http_test_server_fixture):
+def test_h1_pool_strategy_mru(http_test_server_fixture):
   """Test connection re-use strategies of the http 1 connection pool.
 
   Test that with the "mru" strategy only the first created connection gets to send requests.
```

Comment:
preexisting but can we define what mru is here in this function comment.  (and in its method, define lru)

Reply:
Done.